### PR TITLE
fix error thrown when space missing before callback arg

### PR DESCRIPTION
**Pull request by Dominik-Petrik**
closes: #359

### CODE REVIEW

File: packages/eslint-plugin-pf-codemods/lib/helpers.js [hunk 1]
```diff
@@ -552,8 +552,9 @@ function addCallbackParam(componentsArray, propMap) {
                       const createRemoveCurrentParamUseFix = (
                         currentUseOfNewParam
                       ) => {
+                        let tokenBeforeCurrentUse = context.getTokenBefore(currentUseOfNewParam)
                         const targetRange = [
-                          currentUseOfNewParam.range[0] - 2,
+                          tokenBeforeCurrentUse.value === "," ? tokenBeforeCurrentUse.range[0] : tokenBeforeCurrentUse.range[1] + 1,
```
Comment (wise-king-sullyman) [inline packages/eslint-plugin-pf-codemods/lib/helpers.js:557]: Will the token before ever not be `,`?

File: packages/eslint-plugin-pf-codemods/lib/helpers.js [hunk 2]
```diff
@@ -552,8 +552,9 @@ function addCallbackParam(componentsArray, propMap) {
                       const createRemoveCurrentParamUseFix = (
                         currentUseOfNewParam
                       ) => {
+                        let tokenBeforeCurrentUse = context.getTokenBefore(currentUseOfNewParam)
```
Comment (thatblindgeye) [inline packages/eslint-plugin-pf-codemods/lib/helpers.js:555]: Nit: can we make this a const instead of let?

**Review by thatblindgeye — CHANGES_REQUESTED** [review]

File: packages/eslint-plugin-pf-codemods/lib/helpers.js (hunk 2 above)
Comment (Dominik-Petrik) [reply packages/eslint-plugin-pf-codemods/lib/helpers.js:555]: Yep, you're right, thanks.

File: packages/eslint-plugin-pf-codemods/lib/helpers.js (hunk 1 above)
Comment (Dominik-Petrik) [reply packages/eslint-plugin-pf-codemods/lib/helpers.js:557]: It probably won't but I wasn't sure.

**Review by thatblindgeye — APPROVED** [review]

**Pull request merged**